src/api-linux.c

```diff
@@ -840,6 +840,28 @@ SEXP ps__kill_if_env(SEXP r_marker, SEXP r_after, SEXP r_pid, SEXP r_sig) {
   return R_NilValue;
 }
 
+SEXP ps__find_if_env(SEXP r_marker, SEXP r_after, SEXP r_pid) {
+  SEXP phandle;
+  pid_t pid = INTEGER(r_pid)[0];
```

Comment:
I wonder if we should start thinking about using `INTEGER_ELT()` for this. Although we would have to define it for older versions of R I guess.

Not that you would really be using an altrep object here anyway...

Reply:
Do the ALTREP docs say anything about this?

Reply:
I don't think directly, it is in the headers, but that is the API you are supposed to access elements so they full vector doesn't have to be realized.

It probably is not yet documented anywhere because they want the ability to change it if needed.